cypress/plugins/index.ts

```diff
@@ -18,3 +18,8 @@ export default async (on, config) => {
   failFast(on, config);
   return config;
 };
+
+module.exports = (on, config) => {
```

Comment:
I don't think this will work actually with the above configuration. 

1. It is using the old style of config initialization and I think it is not supported with our current cypress ts config
2. It tries to override the above correct config

Simply try to add this line to the above function.

Reply:
You're right, that was stupid. 🙈